thrimbletrimmer/scripts/IO.js

```diff
@@ -0,0 +1,168 @@
+//var desertBusStart = new Date("1970-01-01T00:00:00Z");
+var desertBusStart = new Date("2019-09-20T18:00:00");
```

Comment:
Is there some way we can pass this in such that it can be defined in the main docker_compose.jsonnet config file?
It seems error prone to have multiple places where this value needs to be set, since they can get out of sync.

eg. I'd imagine this file gets minified at some point. Can we pass values into the minifier to set at "build time" to get baked into the minified file?

Alternately, we get `desertBusStart` from a thrimshim API call, or even just as part of the existing response objects.

Reply:
The 100% is not ideal, and I very much don't like it. It's just a value I needed to set somehow. 

Adding it in as a thrimshim API call would be the best way to do it right now; but preventing these kinds of extra round trips is also part of why I'd like to integrate thrimshim and thrimbletrimmer directly in a future iteration.

There's no build process at the moment; but that would certainly be another way to handle it.

Setting the stream Thrimbletrimmer is supposed to be asking for is another issue that faces the same problems.

Reply:
Seems to me that having thrimshim return it as an extra field in its existing responses should be an easy solution without adding an extra round trip?

Reply:
Yup it can be easily added to `get_row()` in `thrimshim`. We can pass in the bus time as a command line option to `thrimshim` unless we want to add a bus_time_start table to the database. Adding it to the database is more work but that way we will get a consistent bus_time_start across all nodes.